source/grid/grid_tools.cc

```diff
@@ -5081,6 +5081,65 @@ namespace GridTools
 #endif // DEAL_II_WITH_MPI
   }
 
+  template <int spacedim>
+  RTree<std::pair<BoundingBox<spacedim>, unsigned int>>
+  build_global_description_tree(
+    const std::vector<BoundingBox<spacedim>> &local_description,
+    MPI_Comm                                  mpi_communicator)
+  {
+#ifndef DEAL_II_WITH_MPI
+    (void)local_description;
+    (void)mpi_communicator;
+    Assert(false,
+           ExcMessage(
+             "GridTools::build_global_description_tree() requires MPI."));
+    return RTree<std::pair<BoundingBox<spacedim>, unsigned int>>{};
```

Comment:
I know I'm coming late to the party, but would it be easy to create an RTree from just the local description here? This would make it easier to run programs with and without MPI...

Reply:
Mmm...so you say it would be better to return a "one-box-tree" for running reasons? That's ok! I can change that :)